services/bitrise/bitrise.service.js

```diff
@@ -18,14 +18,44 @@ export default class Bitrise extends BaseJsonService {
     queryParamSchema,
   }
 
-  static examples = [
-    {
-      title: 'Bitrise',
-      namedParams: { appId: '3ff11fe8457bd304', branch: 'master' },
-      queryParams: { token: 'lESRN9rEFFfDq92JtXs_jw' },
-      staticPreview: this.render({ status: 'success' }),
+  static openApi = {
+    '/bitrise/{appId}': {
+      get: {
+        summary: 'Bitrise',
+        parameters: [
+          pathParam({
+            name: 'appId',
+            example: '4a2b10a819d12b67',
+          }),
+          queryParam({
+            name: 'token',
+            example: 'abc123def456',
+            required: true,
+          }),
+        ],
+      },
+    },
+    '/bitrise/{appId}/{branch}': {
+      get: {
+        summary: 'Bitrise (branch)',
+        parameters: [
+          pathParam({
+            name: 'appId',
+            example: 'e736852157296019',
+          }),
+          pathParam({
+            name: 'branch',
+            example: 'master',
+          }),
+          queryParam({
+            name: 'token',
+            example: 'abc123def456',
```

Comment:
were you planning to update the examples too, or only the tests?

Reply:
Only the tests, that's what I asked for in the linked issue.